rlclientlib/logger/logger_facade.cc

```diff
@@ -11,28 +11,37 @@ namespace reinforcement_learning {
       RETURN_ERROR_ARG(nullptr, status, protocol_not_supported, "Current protocol version is not supported");
     }
 
+    template<typename T>
+    i_async_batcher<T>* create_async_batcher(const utility::configuration& c, i_message_sender* sender, utility::watchdog& watchdog, error_callback_fn* perror_cb) {
+      return new async_batcher<T, fb_collection_serializer>(
+        sender,
+        watchdog,
+        perror_cb,
+        c.get_int(name::INTERACTION_SEND_HIGH_WATER_MARK, 198 * 1024),
```

Comment:
this factory method does not seem INTERACTION specific.
Do we want to fix https://github.com/VowpalWabbit/reinforcement_learning/issues/172 before?

Reply:
uggg, yeah, this won't work as is. I'll take a run at that issue.

Reply:
PR rebased on top of master with the batcher config cleanup.